features/product/viewing_products/not_seeing_lowest_price_for_product_that_has_taxon_excluded_on_channel.feature

```diff
@@ -6,35 +6,37 @@ Feature: Not seeing the lowest price for a product that has a taxon excluded on
 
     Background:
         Given the store operates on a single channel in "United States"
-
         And the store classifies its products as "Category"
         And the "Category" taxon has children taxons "Groceries" and "Special offers"
         And the "Groceries" taxon has child taxon "Vegetables"
-        And the store has a product "Broccoli" priced at "$20.00" belonging to the "Vegetables" taxon
+        And the store has a product "Broccoli" priced at "$20.00"
```

Comment:
I have rearranged these steps to speed up development, I will restore them and implement such steps

Reply:
https://github.com/Sylius/Sylius/pull/14904/files#diff-4c4335cbd32c6b6e36bd44abc5648d20b1f3efd6f0602a4278f1fcf638ce2511L12-R15